static/app/bootstrap/initializeSdk.tsx

```diff
@@ -120,6 +121,43 @@ export function initializeSdk(config: Config, {routes}: {routes?: Function} = {}
     Sentry.setTag('customerDomain.subdomain', customerDomain.subdomain);
   }
 
+  const rawPayload = localStorage.getItem('customerDomainRedirect');
+  localStorage.removeItem('customerDomainRedirect');
+  if (rawPayload) {
+    try {
+      const payload = JSON.parse(rawPayload);
+      const {
+        fullRoute,
+        orglessSlugRoute,
+        redirectOrgURL,
+        routeParams,
+        wrongOrgId,
+        paramOrgId,
+      } = payload;
+      const _customerDomain = payload.customerDomain;
+      // Emit Sentry message for this redirect.
+      // We do this so we can know where we need fix links that makes use of :orgId in the customer domain world.
+      Sentry.setTag('isCustomerDomain', 'yes');
+      Sentry.setTag('customerDomain.organizationUrl', _customerDomain.organizationUrl);
+      Sentry.setTag('customerDomain.sentryUrl', _customerDomain.sentryUrl);
+      Sentry.setTag('customerDomain.subdomain', _customerDomain.subdomain);
+      Sentry.setTag('customerDomain.fromRoute', fullRoute);
+      Sentry.setTag('customerDomain.redirectRoute', orglessSlugRoute);
+      Sentry.setTag('customerDomain.wrongOrgId', wrongOrgId);
+      Sentry.setTag('customerDomain.paramOrgId', paramOrgId);
+      Sentry.setContext('customerDomain', {
+        customerDomain: _customerDomain,
+        fullRoute,
+        orglessSlugRoute,
+        redirectOrgURL,
+        routeParams,
+      });
```

Comment:
Using context is great, do we need tags & context?  Are there tags that we could skip if we also have context data?